gamedata.py

```diff
@@ -720,8 +721,28 @@ def _get_cards(self) -> Generator[tuple[str, dict[str, str]], None, None]:
             except KeyError:
                 meta = {"CHARACTER": "Special"}
             yield card, meta
+    
+    @property
+    @abstractmethod
+    def removals(self) -> list[str]:
+        raise NotImplementedError
+
+    def _get_removals(self) -> Generator[tuple[str, dict[str, str]], None, None]:
+        for card in self.removals:
+            try:
+                meta = get_card_metadata(card)
+            except KeyError:
+                meta = {"CHARACTER": "Special"}
+            yield get_card(card), meta
+        return
+
+    def master_deck_as_html(self) -> Generator[str, None, None]:
+        return self._cards_as_html(self._get_cards())
+
+    def removals_as_html(self) -> Generator[str, None, None]:
+        return self._cards_as_html(self._get_removals())
 
-    def cards_as_html(self) -> Generator[str, None, None]:
+    def _cards_as_html(self, cards: Generator[tuple[str, dict[str, str]], None, None]) -> Generator[str, None, None]:
```

Comment:
The `cards` type hint should just be `Iterable[tuple[str, dict[str, str]]]` -- a generator is iterable, and it's better to assume what is actually used (aka an iterable).